Gemfile

```diff
@@ -6,12 +6,6 @@ ruby '3.1.3'
 # Bundle edge Rails instead: gem "rails", github: "rails/rails", branch: "main"
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/225282562-05b42209-0fa6-4d1e-be94-e46f85d52ed2.png)

**[Optional]**

- [ ] Kindly remove this **NOTE** because this is just an instruction and making your readme file not look clean :fearful: Kindly remove this **NOTE** since you already have a license file for your project :+1: